controllers/comments.controllers.js

```diff
@@ -0,0 +1,14 @@
+const { selectCommentsByArticleId } = require('../models/comments.models');
+
+const getAllCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
```

Comment:
You will need a way to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.

For this think about using Promise.all()

Reply:
Done